galleries/tutorials/index.rst

```diff
@@ -0,0 +1,159 @@
+.. _tutorials:
+
+Tutorials
+=========
+
+This page contains a few tutorials for using Matplotlib.  For the old tutorials, see :ref:`below <user_guide_tutorials>`.
+
+For shorter examples, see our :ref:`examples page <examples-index>`.
+You can also find :ref:`external resources <resources-index>` and
+a :ref:`FAQ <faq-index>` in our :ref:`user guide <users-guide-index>`.
+
+
+.. raw:: html
+
+    <div class="sphx-glr-thumbnails">
+
+
+.. raw:: html
+
+    <div class="sphx-glr-thumbcontainer" tooltip="An introduction to the pyplot interface.  Please also see quick_start for an overview of how Ma...">
+
+.. only:: html
+
+  .. image:: /tutorials/images/thumb/sphx_glr_pyplot_thumb.png
+    :alt: Pyplot tutorial
+
+  :ref:`sphx_glr_tutorials_pyplot.py`
+
+.. raw:: html
+
+      <div class="sphx-glr-thumbnail-title">Pyplot tutorial</div>
+    </div>
+
+
+.. raw:: html
+
+    <div class="sphx-glr-thumbcontainer" tooltip="A short tutorial on plotting images with Matplotlib.">
+
+.. only:: html
+
+  .. image:: /tutorials/images/thumb/sphx_glr_images_thumb.png
+    :alt: Image tutorial
+
+  :ref:`sphx_glr_tutorials_images.py`
+
+.. raw:: html
+
+      <div class="sphx-glr-thumbnail-title">Image tutorial</div>
+    </div>
+
+
+.. raw:: html
+
+    <div class="sphx-glr-thumbcontainer" tooltip="This tutorial aims to show the beginning, middle, and end of a single visualization using Matpl...">
+
+.. only:: html
+
+  .. image:: /tutorials/images/thumb/sphx_glr_lifecycle_thumb.png
+    :alt: The Lifecycle of a Plot
+
+  :ref:`sphx_glr_tutorials_lifecycle.py`
+
+.. raw:: html
+
+      <div class="sphx-glr-thumbnail-title">The Lifecycle of a Plot</div>
+    </div>
+
+
+.. raw:: html
+
+    <div class="sphx-glr-thumbcontainer" tooltip="Using Artist objects to render on the canvas.">
+
+.. only:: html
+
+  .. image:: /tutorials/images/thumb/sphx_glr_artists_thumb.png
+    :alt: Artist tutorial
+
+  :ref:`sphx_glr_tutorials_artists.py`
+
+.. raw:: html
+
+      <div class="sphx-glr-thumbnail-title">Artist tutorial</div>
+    </div>
+
+
+.. raw:: html
+
+    </div>
+
+
+.. toctree::
+   :hidden:
+
+   /tutorials/pyplot
+   /tutorials/images
+   /tutorials/lifecycle
+   /tutorials/artists
+
+.. only:: html
+
+  .. container:: sphx-glr-footer sphx-glr-footer-gallery
+
+    .. container:: sphx-glr-download sphx-glr-download-python
+
+      :download:`Download all examples in Python source code: tutorials_python.zip </tutorials/tutorials_python.zip>`
+
+    .. container:: sphx-glr-download sphx-glr-download-jupyter
+
+      :download:`Download all examples in Jupyter notebooks: tutorials_jupyter.zip </tutorials/tutorials_jupyter.zip>`
+
+
+
+.. _user_guide_tutorials:
+
+User guide tutorials
+--------------------
+
+Many of our tutorials were moved from this section to :ref:`users-guide-explain`:
+
+Introductory
```

Comment:
IMHO, we do not need to list the moved ones individually. There's a reason they are not in the tutorial section, anymore, so the top-level link given above should be enough. - May be ok to keep for a short transition period.